packages/berry-cli/sources/cli.ts

```diff
@@ -38,6 +39,25 @@ function runBinary(path: PortablePath) {
   }
 }
 
+async function printAvailableScripts(configuration: Configuration) {
+  const stdout = process.stdout;
+  const report = await StreamReport.start({configuration, stdout}, async (report: StreamReport) => {
+    const {workspace} = await Project.find(configuration, configuration.startingCwd);
```

Comment:
The `Project.find` (and subsequent `WorkspaceRequiredError` check) should be done outside of the reporter.